02_tidyverse.qmd

```diff
@@ -1,9 +1,10 @@
 ---
-title: "Introduction to the tidyverse"
+title: "Introduction to the Tidyverse"
 format: 
   html:
     toc: true
     code-line-numbers: true
+    embed-resources: true
```

Comment:
I think `embed-resources: true` causes issues with GitHub pages for Quarto books. There is a reason I didn't embed resources in the existing book (even if I am not 100% sure why).

Reply:
I am happy to remove this! Just the extraneous `[chapter-name]_files/` files are really annoying. Do you have any suggestions on how to avoid `embed-resources: true` whiles still avoiding those?

Reply:
They should disappear if you run `quarto render` in the terminal? They only stay when rendering fails I believe.